docs/security/unlock-user-accounts.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Unlock user accounts from the Clerk Dashboard
+description: Use the Clerk Dashboard to unlock user accounts.
+---
+
+# Unlock user accounts from the Clerk Dashboard
+
+<Callout type="info">
+This feature is applicable to user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) or [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
+</Callout>
+
+Users with admin roles can override lockouts through the Clerk dashboard.
+
+1. In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **[Users](https://dashboard.clerk.com/last-active?path=users)**.
+2. Locate the locked user by looking for a "locked" badge next to their username/email. 
+3. Open the admin menu (three little dots).
+4. Select "Unlock" to unlock the user account.
+
+<Images
+width={1512}
+height={914}
+src="/docs/images/security/userlock_dashboard.png"
+alt=" "
+/>
+
+Alternatively, you can unlock a user's account on their profile page by scrolling to the "Unlock user" section and selecting "Unlock user."
```

Comment:
This is missing the step where the application owner must follow all of the above steps, and then click on "View profile", and then can scroll to the "Unlock user" section where they can select **Unlock user**.